main/gcc/APKBUILD

```diff
@@ -171,6 +171,7 @@ source="https://gcc.gnu.org/pub/gcc/releases/gcc-${_pkgbase:-$pkgver}/gcc-${_pkg
 	320-libffi-gnulinux.patch
 
 	gcc-pure64-mips.patch
+	0016-invalid_tls_model.diff
```

Comment:
I think we need to rename it to `0016-invalid_tls_model.patch` or it will not be applied by abuild.

Reply:
Derp, right. Thanks for noticing